content/process/change/secrets_managment.md

```diff
@@ -14,12 +14,36 @@ areas:
 ## Background
 
 Secrets must be stored in a secure way, and a documented in a central place.
-[Cryptographic failures are the second highest risk in the OWASP top ten](https://owasp.org/Top10/A02_2021-Cryptographic_Failures/) so rigor and process is essential.
+[Cryptographic failures are the second highest risk in the OWASP top ten](https://owasp.org/Top10/A02_2021-Cryptographic_Failures/) 
+so rigor and process is essential.
 
 {{< figure src="/images/secrets-management.svg" alt="Change Records" >}}
 
 ## How we implement this control
 
-* We use AWS secrets manager to store infrastructure secrets
-* Secrets are provisioned in our terraform model ([instructions here](https://github.com/kosli-dev/knowledge-base/blob/master/add_secrets.md))
-* Secrets are entered via the AWS cloud console by the authorized team members
\ No newline at end of file
+### Infrastructure secrets
+* We use AWS secrets manager to store infrastructure secrets.
+* Infrastructure secrets are handled with a separate [terraform-server repository](https://github.com/kosli-dev/terraform-server)
```

Comment:
I believe these repos are private and assume this info will be public?

Reply:
Yes the kosli-sdlc repo is public, and the link is private. My assumption is that this is a process that shall be read by the Kosli developers so that they understand the process and follow it.